presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaConnectorConfig.java

```diff
@@ -79,18 +71,6 @@ public KafkaConnectorConfig setDefaultSchema(String defaultSchema)
         return this;
     }
 
-    public List<HostAddress> getNodes()
-    {
-        return nodes;
-    }
-
-    @Config("kafka.nodes")
-    public KafkaConnectorConfig setNodes(String nodes)
-    {
-        this.nodes = (nodes == null) ? null : parseNodes(nodes).asList();
-        return this;
-    }
-
```

Comment:
Just wanna double confirm if this would break the existing config.  is there any config migration required for this change?

Reply:
The existing config can work as-is, no config migration is needed.  
The reason is default cluster metadata supplier(FileKafkaClusterMetadataSupplier) will load FileKafkaClusterMetadataSupplierConfig which is the copy of the config that has been removed from this file.

Reply:
Cool, sounds good to me.